ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -98,17 +98,17 @@ - (void)tokensButtonPressed:(id)sender {
 }
 
 - (void)overridesButtonPressed:(id)sender {
-    MSFFluentTheme *fluentTheme = [[self view] fluentTheme];
-    [fluentTheme setProviderWithProvider:[ObjectiveCDemoColorProviding2 alloc]];
+    [[self view] setColorProvider:[[ObjectiveCDemoColorProviding2 alloc] init]];
```

Comment:
Note: you should _never_ call `alloc` on an Objective-C object without also calling `init` -- it's the same as calling `malloc` in C without actually initializing the allocated memory. It _might_ be clean, usable memory, or it might be junk that immediately crashes when you access it.